resume/src/App.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Section from "./components/Section";
+import resumeJSON from "./data/resume";
+import Basics from "./components/sections/Basics";
+import Work from "./components/sections/Work";
+
+import "./App.css";
+
+function App() {
+  let counter = 0;
+  const sections = Object.keys(resumeJSON).map(section => {
+    counter++;
+    return <Section title={section} info={resumeJSON[section]} key={counter} />;
+  });
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <p>Resume</p>
```

Comment:
Probably best to be using an H1 for this, assuming it is the title of the page for the best semantic code.

Reply:
Quick counterpoint: H1 does have SEO implications. Using a `p` can be fine if you're saving your `h1` for, say, your full name, or your title. :-)